### ISSUE TITLE
Further clean up build process

### ISSUE THREAD

**Issue by benstoltz** [Esri/cedar#182]
Investigate whether there are other optimizations / cleaning up of build system.
Would Rollup be a good option? 

@tomwayson

**Comment by tomwayson** [comment]
Looking over the gruntfile, nothing jumps out at me as being extraneous. 

#181 is obviously going to introduce new build steps, but it's probably easiest to bring them into the existing (grunt) pipeline rather than switch to gulp or npm scripts just b/c grunt is so 2014.

Rollup will mainly help if the source code:
a. uses ES2015 style imports/exports (ES2015+ or TypeScript)
b. is split amongst multiple modules

Another possible way Rollup could help is by inlining the default chart definitions through it's [JSON plugin](https://github.com/rollup/rollup-plugin-json), if that's desirable, and ultimately, once tree-shaking works a little better, and Cedar's deps (vega/d3) are written in imports/exports, then something like Rollup or Webpack could theoretically produce a build w/ just the code needed by Cedar.

Basically, if #181 breaks the source into modules, then we should probably look at Rollup.

**Comment by benstoltz** [comment]
Closed by #222 nearly all tooling was reworked.
